Sources/PrimerSDK/Classes/Core/Primer/AppState.swift

```diff
@@ -26,10 +26,12 @@ internal protocol AppStateProtocol: class {
     var authorizationToken: String? { get set }
     var customerToken: String? { get set }
     var sessionId: String? { get set }
+    
+    func setApayaResult(_ result: Result<ApayaWebViewResult, ApayaException>)
```

Comment:
Are those for testing reasons? Or will the app state have those functions?

Reply:
I'm more and more leaning towards this approach, and in the long run we would split up the app state in more manageable stores. The reason is that it simply becomes more testable + the internal workings of the app state becomes the preserve of that class alone. The rest of the SDK shouldn't "see" everything inside this class.